pkg/kubelet/cm/cpumanager/cpu_manager_test.go

```diff
@@ -120,6 +120,10 @@ func (p *mockPolicy) GetPodTopologyHints(s state.State, pod *v1.Pod) map[string]
 	return nil
 }
 
+func (p *mockPolicy) GetAssignableCPUs(m state.State) cpuset.CPUSet {
+	return cpuset.NewCPUSet()
+}
+
```

Comment:
Shouldn't this be `GetAllocatableCPUs()`?

Reply:
yes, forgot to update/lost in rebase. Will update.